study-builder/studies/pancan/snippets/question-primary-cancer-list-self.conf

```diff
@@ -55,6 +55,23 @@
           }
         ]
       }
+    },
+    {
+      "ruleType": "UNIQUE",
+      "allowSave": false,
```

Comment:
allowSave is a non-nullable attribute in table: validation with a default value: false 
I see some notes in PR description reg its usage.
Want to clarify that if the value is NOT set for any validation, its defaulting to "false" .
If you're expecting front-end to check this value and behave differently I don't think the approach will work.

Reply:
If front-end behavior is same as every other validation which has default value of "false", we can remove explicitly setting it to false here.